packages/extension-sample/README.md

```diff
@@ -0,0 +1,68 @@
+ __                       _          __      _                 _             
```

Comment:
This is a rough example of what an extension looks like. For now we use this extension in the `typescript-minimal` project, but in the future we can decide to either remove it, or use it as a starting point for the generator to allow partners to create an application extension.